### PR TITLE
Feature/new design

### PR DESCRIPTION
**Pull request by LucaMilanese90**
New design for menubar and browse page.

### CODE REVIEW

File: src/components/molecules/Menu.tsx [hunk 1]
```diff
@@ -38,9 +38,10 @@ export default function Menu(): ReactElement {
   return (
     <nav className={styles.menu}>
       <Link to="/" className={styles.logo}>
-        <Logo branding />
+        <Logo noWordmark />
```
Comment (moritzkirstein) [inline src/components/molecules/Menu.tsx:41]: again have a look at the `branding` implementation please. I think this is required

File: src/components/molecules/UserPreferences/index.tsx [hunk 2]
```diff
@@ -20,15 +20,15 @@ export default function UserPreferences(): ReactElement {
         <ul className={styles.preferencesDetails}>
           <Currency />
           <TokenApproval />
-          <Appearance darkMode={darkMode} />
+          {/* <Appearance darkMode={darkMode} /> */}
```
Comment (moritzkirstein) [inline src/components/molecules/UserPreferences/index.tsx:23]: Just delete this I'd say. Also you can delete all darkmode stuff in this file

File: src/components/organisms/AssetActions/Pool/Graph.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ import styles from './Graph.module.css'
 declare type GraphType = 'liquidity' | 'price'
 
 // Chart.js global defaults
-defaults.global.defaultFontFamily = `'Overpass', -apple-system, BlinkMacSystemFont,
+defaults.global.defaultFontFamily = `'Inter', -apple-system, BlinkMacSystemFont,
```
Comment (moritzkirstein) [inline src/components/organisms/AssetActions/Pool/Graph.tsx:27]: Nice that you found this :) 👍

File: src/components/organisms/AssetActions/Pool/TokenList.module.css [hunk 4]
```diff
@@ -15,7 +15,7 @@
 }
 
 .tokenlist.highlight {
-  background: var(--background-highlight);
+  /* background: var(--background-content); */
```
Comment (moritzkirstein) [inline src/components/organisms/AssetActions/Pool/TokenList.module.css:18]: please remove comments

File: src/components/organisms/AssetActions/Trade/TradeInput.module.css [hunk 5]
```diff
@@ -3,7 +3,7 @@
   padding: var(--spacer) calc(var(--spacer) * 2);
   margin-left: -2rem;
   margin-right: -2rem;
-  background: var(--background-highlight);
+  /* background: var(--background-content); */
```
Comment (moritzkirstein) [inline src/components/organisms/AssetActions/Trade/TradeInput.module.css:6]: comments if not needed

File: src/global/Styles.tsx [hunk 6]
```diff
@@ -1,9 +1,9 @@
 import React, { ReactElement, ReactNode } from 'react'
 import { ToastContainer } from 'react-toastify'
 
-import '@fontsource/overpass/400.css'
-import '@fontsource/overpass/700.css'
-import '@fontsource/overpass/900.css'
+import '@fontsource/inter/400.css'
+import '@fontsource/inter/700.css'
+import '@fontsource/inter/900.css'
```
Comment (moritzkirstein) [inline src/global/Styles.tsx:6]: @oceanByte do these weights fit with forever branding?

File: src/global/_variables.css [hunk 7]
```diff
@@ -12,53 +14,57 @@
...
+  --font-size-h3: 2rem;
+  --font-size-h4: 1.5rem;
+  --font-size-h5: 1.2rem;
+  --font-weight-light: 300;
   --font-weight-base: 400;
-  --font-weight-bold: 700;
-  --font-weight-heavy: 900;
+  --font-weight-semibold: 500;
+  --font-weight-bold: 600;
+  --font-weight-heavy: 700;
```
Comment (moritzkirstein) [inline src/global/_variables.css:63]: please check with imported weights. If possible lets use 3 different weights at max

**Review by moritzkirstein — CHANGES_REQUESTED** [review]
Please have a look at the searchbar on mobile screens. Needs some fixing

Regarding darkmode: We will need some fixing here. On systems with darkmode configured the market loads with darkmode colors. One quick fix could be to remove the .dark associated stylings in the _variables.css

I really enjoy the styling! The market looks great 🚀

File: src/components/atoms/Logo.tsx [hunk 8]
```diff
@@ -1,24 +1,17 @@
...
 
 import styles from './Logo.module.css'
 
 export default function Logo({
-  noWordmark,
-  branding
+  noWordmark
 }: {
   noWordmark?: boolean
-  branding?: boolean
```
[the remark this answers is not in the document]
Comment (LucaMilanese90) [reply src/components/atoms/Logo.tsx:13]: I don't see any file other than Menu.tsx where `branding` is passed as a parameter.

**Review by moritzkirstein — APPROVED** [review]
Looks good! This is something to present and show in the upcoming mtg and gather feedback!

One small remark: I noticed the cookie banner still has a black background if the user OS has darkmode enabled. I'll open an issue on this

**Pull request merged**